gateway/cli/Cargo.toml

```diff
@@ -23,6 +23,7 @@ anyhow = { workspace = true }
 bitcoin = { workspace = true }
 clap = { workspace = true }
 clap_complete = "4.5.37"
+fedimint-client = { path = "../../fedimint-client", version = "=0.5.0-alpha", default-features = false }
```

Comment:
Though: maybe we should split the log functionality into own crate so we don't need to import the whole client here?

Reply:
Sounds like a good idea to me. Made an issue https://github.com/fedimint/fedimint/issues/6358